### PR TITLE
breaking(theme): removing brand namespace

### PR DESCRIPTION
**Pull request by dbanksdesign**
#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] Relevant documentation is changed or added (and PR referenced)
- [x] `yarn test` passes and tests are updated/added
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/ui/src/theme/createTheme.ts [hunk 1]
```diff
@@ -23,6 +24,17 @@ const setupToken: SetupToken<WebDesignToken> = ({ token, path }) => {
   return { name, original, path, value, toString: () => `var(${name})` };
 };
 
+function applyBrandColor({ tokens, value, key }) {
```
Comment (calebpollman) [inline packages/ui/src/theme/createTheme.ts:27]: Kind of confused by the name here 😅

Comment (dbanksdesign) [reply packages/ui/src/theme/createTheme.ts:27]: it _applies_ the brand color 😜

File: packages/ui/src/theme/__tests__/createTheme.test.ts [hunk 2]
```diff
@@ -304,5 +304,19 @@ describe('@aws-amplify/ui', () => {
...
       });
     });
+
+    it('should apply primaryColor', () => {
+      const { tokens } = createTheme({
+        name: 'test',
+        primaryColor: 'red',
+        secondaryColor: 'blue',
+      });
+      expect(tokens.colors.primary[10].value).toEqual(
```
Comment (reesscot) [inline packages/ui/src/theme/__tests__/createTheme.test.ts:314]: should this test check that all the color values are set?

**Comment by reesscot** [comment]
Should the migration guide have a note on this?

**Comment by dbanksdesign** [comment]
The migration docs are in there: https://github.com/aws-amplify/amplify-ui/pull/4635/files#diff-c33f27a7d184b39a7b5c2c54004d873ce26486002d9b1c5c9a4fab2224cf49de

**Review by reesscot — APPROVED** [review]

File: packages/ui/src/theme/types.ts [hunk 3]
```diff
@@ -127,6 +133,8 @@ export interface DefaultTheme extends Pick<Theme, 'name' | 'overrides'> {
  */
 export interface WebTheme
   extends Pick<DefaultTheme, 'breakpoints' | 'name' | 'overrides'> {
+  primaryColor?: string;
+  secondaryColor?: string;
```
Comment (calebpollman) [inline packages/ui/src/theme/types.ts:137]: Can we also `Pick` these from `DefaultTheme`? Can be done in a follow up

**Review by calebpollman — APPROVED** [review]

**Pull request merged**